src/test/resources/features/QuestionBank/QuestionBank.feature

```diff
@@ -0,0 +1,31 @@
+@QuestionBank @Regression
+Feature: Question Bank
+
+  Background:
+    Given user prepare question bank request
+
+  @Negative @QuestionBank
+  Scenario: Create question bank without logging in
+    And user prepare auth request
+    When user hit logout endpoint
+    And user hit create question bank endpoint with title "QA Title" and description "QA Description"
+    Then question bank response code should be 401
+
+  @Negative @QuestionBank
+  Scenario: Create question bank with empty title and description after logging in as admin
+    And user prepare auth request
+    When user do login with email "admin@admin.com" and password "administratorfunctionapp"
+    And user hit create question bank endpoint with title "" and description ""
+    Then question bank response code should be 400
+    And question bank error response has key "title" and value "NotBlank"
+    And question bank error response has key "description" and value "NotBlank"
```

Comment:
Add `And user hit logout endpoint` after this line (think as teardown) for this scenario, so the login data is not persisted in other scenarios (due to use of autowired).

Some feature files still do not use this, but in future tests will be corrected